packages/lifi-adapter/dev/server.mjs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+import http from 'node:http';
+import { buildSnapshot } from './libDemo.mjs';
+
+const PORT = Number(process.env.DEMO_PORT || 3001);
+const MIN_INTERVAL_MS = Number(process.env.RATE_LIMIT_MIN_TIME_MS || 200);
+
+let lastRequestAt = 0;
+
+const server = http.createServer(async (req, res) => {
+  if (req.method === 'GET' && req.url === '/snapshot') {
```

Comment:
Good job being scrappy and figuring out a way to test it yourself!

I will improve this in the original demo.